pipparse.rb

```diff
@@ -9,4 +9,4 @@
 until STDIN.eof? do
   message = Pipboy::Parser.parse_message(STDIN)
   STDOUT.puts(message.to_json)
-end
+end
```

Comment:
Make sure 
```
"trim_trailing_white_space_on_save": true,
"ensure_newline_at_eof_on_save": true
```

are both set in your sublime config so that there isn't extra trailing white space a tthe end of lines. It also makes sure we end up with newlines at the end of a file